src/app/index.html

```diff
@@ -270,10 +270,10 @@ <h3 class="bullet-header">
               <div class="card-image col-sm-8">
                 <!--<img src="../assets/images/icon-dependencies.png" />-->
                 <h3>
-                  Allow development teams be more confident in the technology decisions they make.
+                  Allow development teams to be more confident in the technology decisions they make.
                 </h3>
                 <p>
-                  OpenShift.io helps developers make better decisions. It provides actionable recommendations and insights around the software components developers use. OpenShift.io Analytics applies machine learning algorithms based on their usage pattern of components. The data is gathered from various public data sources such as Github, Maven and NPM along with our own internal OpenShift data. Through this comprehensive dataset, developers get tangible information for credible
+                  OpenShift.io helps developers make better decisions. It provides actionable recommendations and insights around the software components developers use. OpenShift.io Analytics applies machine learning algorithms based on their usage pattern of components. The data is gathered from various public data sources such as Github, Maven and NPM along with our own internal OpenShift data.
```

Comment:
Last sentence should be " Through this comprehensive dataset, developers get tangible information for credible decision-making."